development/templates/development/my_bots.html

```diff
@@ -2,17 +2,30 @@
 
 {% block page_content %}    
 <div class="container">
-    <h1>My Bots</h1> 
-
-    {% load render_table from django_tables2 %}
-    <body>
-        {% render_table table %}
-    </body>
-</div>       
-
-
-<hr>
-<div class="form-row text-center">
+    <h1>My Bots</h1>
+    <div class="pre-scrollable">
+        <table 
+            class="table table-striped table-bordered table-sm">
+          <thead>
+            <tr class="bg-primary text-white" style="
+            position: sticky;
+            top: 0;">
+              <th>Name</th>
+              <th>Token</th>
+              <th></th>
+            </tr>
+          </thead>
+          <tbody>
+            {% for bot in object_list %}
+            <tr>
+              <td width="21%">{{ bot.name }}</td>
+              <td width="73%" style="word-break: break-all">{{ bot.token}}</td>
```

Comment:
I know you don't like css 😂 but this is very hardcoded and it will work only screens like yours

Reply:
Now it uses a class instead of a style.